owncloudApp/src/main/java/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.kt

```diff
@@ -332,57 +477,59 @@ class ErrorMessageAdapter {
                     return f.format(R.string.auth_account_not_the_same)
                 RemoteOperationResult.ResultCode.OK_REDIRECT_TO_NON_SECURE_CONNECTION ->
                     return f.format(R.string.auth_redirect_non_secure_connection_title)
-                else -> if (result.httpPhrase != null && result.httpPhrase.length > 0)
-                    return result.httpPhrase
+//                else -> if (result.httpPhrase != null && result.httpPhrase.length > 0)
+//                    return result.httpPhrase
             }
 
-            return getGenericErrorMessageForOperation(operation, result, res)
+            return getGenericErrorMessageForOperation(operation, resultCode, res)
         }
 
         /**
          * Return a user message corresponding to a generic error for a given operation.
          *
-         * @param operation     Operation performed.
-         * @param res           Reference to app resources, for i18n.
+         * @param operationType [OperationType] performed.
+         * @param resultCode    ResultCode of a [RemoteOperation] performed.
          * @return User message corresponding to a generic error of 'operation'.
          */
         private fun getGenericErrorMessageForOperation(
-            operation: RemoteOperation<*>?,
-            result: RemoteOperationResult<*>,
+            operationType: OperationType,
+            resultCode: RemoteOperationResult.ResultCode?,
             res: Resources
         ): String {
             val f = Formatter(res)
 
-            if (operation is UploadFileOperation)
-                return f.format(
-                    R.string.uploader_upload_failed_content_single,
-                    operation.fileName
-                )
-            if (operation is DownloadFileOperation)
-                return f.format(
-                    R.string.downloader_download_failed_content,
-                    File(operation.savePath).name
-                )
-            if (operation is RemoveFileOperation) return f.format(R.string.remove_fail_msg)
-            if (operation is RenameFileOperation) return f.format(R.string.rename_server_fail_msg)
-            if (operation is CreateFolderOperation) return f.format(R.string.create_dir_fail_msg)
-            if (operation is CreateShareViaLinkOperation || operation is CreateShareWithShareeOperation)
+//            if (operation == OperationType.UPLOAD_FILE)
```

Comment:
delete commented code please. Git helps you in case you want to recover it ;-)

Reply:
Done